Main.hs

```diff
@@ -50,24 +53,29 @@ main =
     runMigration migrateTables
     liftIO loop
 
+selectPreviousLogItem :: MonadIO m => SqlReadT m [Item]
+selectPreviousLogItem = do
+  items <- select $ from $ \li -> do
+            orderBy [desc (li ^. LogItemId)]
+            limit 1
+            return (li ^. LogItemId, li ^. LogItemTitle)
+  return $ fmap (uncurry Item . bimap unValue unValue) items
```

Comment:
This does not meet the requirements specified in: https://stackoverflow.com/q/50143910/750216
`Is there any way in esqueleto to project a subset of the columns to custom records (different than the entity) instead of tuples? This being done without an extra projection from tuples to custom records.`